src/util/event-emitter.js

```diff
@@ -21,14 +19,14 @@ export class EventEmitter{
   }
 
   next(value){
-    this._subject.onNext(value);
+    this._subject.next(value);
   }
 
   throw(error){
-    this._subject.onError(error);
+    this._subject.error(error);
   }
 
   return(){
-    this._subject.onCompleted();
+    this._subject.complete();
   }
 }
```

Comment:
And there is even a PR from Hannah to remove RX completely to reduce the size to something tiny https://github.com/ngUpgraders/ng-forward/pull/25. I'm going to check it out now. 

Pete, do you know how to merge this PR into my local repo?